pages/checkout/_need-help.html

```diff
@@ -5,4 +5,14 @@ <h2 class="need-help-text">Need help deciding?</h2>
       >Contact Sales</a
     >
   </div>
+<div class="row row-page-header padding-top-none">
```

Comment:
Could you put this below and outside the "Need help deciding?" box? The same way it is below and outside the box on the `/pricing` page?

Reply:
Whoops, great catch. I don't have a great local setup for this probably should get that going!